src/game/client/components/menus_browser.cpp

```diff
@@ -501,7 +501,7 @@ int CMenus::DoBrowserEntry(const void *pID, CUIRect *pRect, const CServerInfo *p
 			if(FilterInfo.m_SortHash&IServerBrowser::FILTER_SPECTATORS)
 				str_format(aTemp, sizeof(aTemp), "%d/%d", pEntry->m_NumPlayers, pEntry->m_MaxPlayers);
 			else
-				str_format(aTemp, sizeof(aTemp), "%d/%d", pEntry->m_NumClients, pEntry->m_MaxClients);
+				str_format(aTemp, sizeof(aTemp), "%d/%d (%d/%d)", pEntry->m_NumClients, pEntry->m_MaxClients, pEntry->m_NumPlayers, pEntry->m_MaxPlayers);
```

Comment:
This isn't very intuitive, is it? Maybe there could be an option in the server browser to toggle between players/max players and clients/max clients?

Reply:
This is the case with `FilterInfo.m_SortHash&IServerBrowser::FILTER_SPECTATORS`.
I don't know how this works though.
In fact, I add this because i find the default NumClient/MaxClient counter intuitive, I personally join server to play, not to watch.
This commit can be dropped anyway

Reply:
Right, so `br_filter_spectators` (ideally with a button on the server browser UI) should take care of that. Changing what the default should be should probably be an issue of its own, although I don't mind the current state of things.